apps/desktop-wallet/posthog-filter-out.json

```diff
@@ -34,5 +34,11 @@
     "type": "string",
     "operator": "is_not",
     "value": "Auto lock"
+  },
+  {
+    "property": "message",
+    "type": "string",
+    "operator": "is_not",
+    "value": "Synching address data"
```

Comment:
I added this to the Posthog Filter Out plugin, so I am committing it here so that we can track it. It will eventually be removed once we confirm that throttling works.